src/app/util/af.h

```diff
@@ -184,6 +184,46 @@ uint16_t emberAfIndexFromEndpointIncludingDisabledEndpoints(chip::EndpointId end
  */
 uint16_t emberAfFindClusterServerEndpointIndex(chip::EndpointId endpoint, chip::ClusterId clusterId);
 
+/**
+ *  @brief Returns an index of the given endpoint the given cluster server
+ *
+ * Returns kEmberInvalidEndpointIndex if the given endpoint does not support the
+ * given cluster or if the given endpoint is disabled.
+ *
+ * Unlike emberAfFindClusterServerEndpointIndex, this function always returns the same index
+ * for a given endpointId instance, static or dynamic.
```

Comment:
```suggestion
 * Unlike emberAfFindClusterServerEndpointIndex, this function always returns the same index
 * for a given endpointId instance, fixed or dynamic, if it does not return kEmberInvalidEndpointIndex.
```

Because an endpoint can _absolutely_ go from returning `kEmberInvalidEndpointIndex` to returning some valid index or the other way.

Also, later on we call the fixed endpoints "fixed", not static.